modules/core/src/main/java/org/apache/ignite/internal/managers/communication/GridIoManager.java

```diff
@@ -3143,4 +3199,29 @@ public long binLatencyMcs() {
             return latencyLimit / (1000 * (resLatency.length - 1));
         }
     }
+
+    /**
+     * @return Security subject id.
```

Comment:
```
@param msg Communication message.
@return A pair that represents a security subject id and security context. The returned value can be {@code null} in case of security context is not enabled.
```

Reply:
fixed.